### ISSUE TITLE
Can't see target of target on many spawned mobs

### ISSUE THREAD

**Issue by fgsfdss** [LightsHope/server#807]
Drakonids on Nefarian, Corpse Scarabs on Anub, Slimes on Grobbulus, Spiderlings on Maexxna, ...
A few off the top of my head, but it's all around the world with spawned creatures specifically.

**Comment by GuybrushGit** [comment]
Addon issue maybe? Tested with spiderlings and they show their target just fine.

**Comment by knasen88** [comment]
This happens to almost every summon creature in the game, felguards in BWL for example. Anything that gets summon in to the fight mid-combat usually behaves this way. rarely they function but usually no target showing.

**Comment by berillis** [comment]
Yes, this is quite irritating when tanking. The only indication I get from the mob I should be tanking if I'm actually tanking it, is to move and see if the mob follows me or check if I take damage. 

Definitely linked to summons that happen mid-combat, happens every time in BWL with the Blackwing Warlock summoned Felguards. Same thing with Gargoyles spawned in Stratholme (by Eye of Naxxramas) and Voidwalkers spawned in Dire Maul North (by the wandering eyes, don't remember the name)

**Comment by thepparker** [comment]
This is pretty longstanding. We tried some hacky changes to get it to work in Naxxramas, but they have proven to be unsuccessful in solving the issue. It needs a second go-around methinks.
